lib/ReactViews/Tools/DiffTool/DiffTool.tsx

```diff
@@ -513,48 +500,36 @@ class Main extends Component<MainPropsType> {
       this.location && datesSelected && this.diffStyle !== undefined;
 
     return (
-      <Text large>
-        <DiffAccordion viewState={viewState} t={t}>
-          <MainPanel
-            isMapFullScreen={viewState.isMapFullScreen}
-            styledMaxHeight={`calc(100vh - ${viewState.bottomDockHeight}px - 230px)`}
-          >
-            {isShowingDiff && (
-              <>
-                <Box centered left>
-                  <BackButton
-                    css={`
-                      color: ${theme.textLight};
-                      border-color: ${theme.textLight};
-                    `}
-                    transparentBg
-                    onClick={this.resetTool}
-                  >
-                    <BoxSpan centered>
-                      <StyledIcon
-                        css="transform:rotate(90deg);"
-                        light
-                        styledWidth="16px"
-                        glyph={GLYPHS.arrowDown}
-                      />
-                      <TextSpan noFontSize>{t("general.back")}</TextSpan>
-                    </BoxSpan>
-                  </BackButton>
-                </Box>
-                <Spacing bottom={3} />
-                <Text medium textLight>
-                  {t("diffTool.differenceResultsTitle")}
-                </Text>
-                <Spacing bottom={2} />
-              </>
-            )}
-            <Text textLight>{t("diffTool.instructions.paneDescription")}</Text>
-            <Spacing bottom={3} />
+      <WorkflowPanel
+        viewState={viewState}
+        title={t("diffTool.title")}
+        icon={GLYPHS.difference}
+        onClose={action(() => {
+          this.resetTool();
+          this.closePanel();
+        })}
+      >
+        <div
+          css={`
+            display: flex;
+            flex-direction: column;
+            height: 100%;
+            padding: 5px 10px;
```

Comment:
nitpick: This would be more consistent with other panels as `padding: 15px`
```suggestion
            padding: 15px;
```